Feature/ae 1108 laskuttaja

muep opened the pull request:
Tarvitsee kaveriksi backend-muutokset tuolta https://github.com/solita/etp-core/pull/511/ (nämä on läpi)

muep committed:
AE-1108 Add modheaders profile for laskuttaja@solita.fi

muep committed:
AE-1108 Add laskuttaja to roles

muep committed:
AE-1108 Add some navigation for laskuttaja

This is initially just identical to what the main user would use.

muep committed:
AE-1108 Disable some fields for Laskuttaja

muep committed:
AE-1108 hide Liitteet for Laskuttaja

hkorpi commented on src/components/Energiatodistus/ET2013Form.svelte:
```diff
@@ -46,6 +46,7 @@
   export let luokittelut;
   export let schema;
   export let disabled = false;
+  export let disabledRt = false;
```
mitä tarkoittaa disabledRt?

muep replied:
disabled rakennustunnus. Tämä varmaan katoaa tai nimeytyy uudestaan nyt kun katsotaan tämä loppuun.

VesaLahd commented on src/components/Energiatodistus/EnergiatodistusForm.svelte:
```diff
@@ -66,6 +67,8 @@
     R.propEq('tila-id', EtUtils.tila.draft, energiatodistus)
   );
 
+  $: disabledRt = Kayttajat.isLaskuttaja(whoami);
```
whoami tulee kakutetusta futuresta, eli sen pitäisi olla ihan staattista tietoa, joka tekee tämän reaktiivisuuden turhaksi

muep replied:
Nykyisessä versiossa tätä ei enää olekaan.

VesaLahd commented on src/components/Energiatodistus/ET2018Form.svelte:
```diff
@@ -151,6 +152,7 @@
   {schema}
   {inputLanguage}
   {disabled}
+  {disabledRt}
```
Itse ehkä antaisin tälle jonkun kuvaavamman isLaskuttaja-tms attribuutin nimen ja syöttäisin sinne suoraan Kayttajat.isLaskuttaja(whoami) tuloksen. Tai sitten avaa tuon disabledRt muotoon disabledRakennustunnus tms. Tämän propdrillaus on imo hieman turhaa kun kaikki vaadittava tuon arvon muodostamiseen passataan jo itsessään tälle tasolle.

muep replied:
Viimeisimmässä versiossa tätä ei tässä muodossa ole. Sen sijaan whoami toimitetaan sinne perustietokomponentille, ja se omin päin disabloi mikäli aihetta on.

MattiNieminen committed:
AE-1108 implement possibility for laskuttaja to send viesti to everyone

muep committed:
AE-1108 avoid the unnecessary disabledRt

Instead, whoami information is provided to the The
RakennuksenPerustiedot component, which can then just decide itself if
it should disable the rakennustunnus field.

muep committed:
Merge remote-tracking branch 'origin/develop' into feature/ae-1108-laskuttaja

muep committed:
Replace ifElse with allPass

Based on our discussion concerning the merge, this seemed to be
condered a nicer way to express the condition.
